superset/assets/javascripts/dashboard/index.jsx

```diff
@@ -8,37 +8,19 @@ import { initEnhancer } from '../reduxUtils';
 import { appSetup } from '../common';
 import { initJQueryAjax } from '../modules/utils';
 import DashboardContainer from './components/DashboardContainer';
-// import rootReducer, { getInitialState } from './reducers';
-
-import emptyDashboardLayout from './v2/fixtures/emptyDashboardLayout';
-import rootReducer from './v2/reducers/';
+import getInitialState from './reducers/initState';
```

Comment:
nit -- I would call this `getInitialState`, it's not clear it's a function (vs an object) and there's no benefit in saving 3 characters by calling it `init` instead of `initial`

Reply:
but love having it in its own file! 💯

Reply:
fixed.